front/lib/api/data_source_view.ts

```diff
@@ -18,14 +18,22 @@ export function filterAndCropContentNodesByView(
       return null;
     }
 
-    // We stop at the first parent that is in the view.
+    // At the first parent that is in the view, we know the content node is in the view
     const indexToSplit = parentInternalIds.findIndex((p) =>
```

Comment:
We should actually update this to go all the way up to the oldest ancestor that is in the view, no? 🤔

Reply:
As per irl :+1: